test/e2e_node/util.go

```diff
@@ -85,6 +86,32 @@ func getCurrentKubeletConfig() (*componentconfig.KubeletConfiguration, error) {
 	return kubeCfg, nil
 }
 
+// Convenience method to set the evictionHard threshold during the current context.
+func tempSetEvictionHard(f *framework.Framework, evictionHard string) {
+	tempSetCurrentKubeletConfig(f, func(initialConfig *componentconfig.KubeletConfiguration) {
+		initialConfig.EvictionHard = evictionHard
+	})
+}
+
+// Must be called within a Context. Allows the function to modify the KubeletConfiguration during the BeforeEach of the context.
+// The change is reverted in the AfterEach of the context.
+func tempSetCurrentKubeletConfig(f *framework.Framework, updateFunction func(initialConfig *componentconfig.KubeletConfiguration)) {
+	var oldCfg *componentconfig.KubeletConfiguration = nil
+	BeforeEach(func() {
+		oldCfg, err := getCurrentKubeletConfig()
+		framework.ExpectNoError(err)
+		newCfg := *oldCfg
```

Comment:
I think I did this correctly, can you review my change?  It seems to work correctly.